### PR TITLE
WIP: Tutorial of the month.

### PR DESCRIPTION
**Pull request by chavid**

### CODE REVIEW

File: CWP/papers/HSF-CWP-2017-02_training/latex/cwp-wg-training-careers.tex [hunk 1]
```diff
@@ -681,6 +681,13 @@ \subsection{Resources and Incentives}
 members of the community is crucial to long-term success of HEP training
 endeavors.
 
+In the context of web-based training, if HSF helps to constitute a living
+collection of online tutorials, we could organize regular events such as
+a "Tutorial of the month", based on students votes. From a more general point
+of view, since the publications are the most universal recognized items
+in the research community, we should try hardly to create some sort of
+training journal, with reviewed training papers.
```
[the remark this answers is not in the document]
Comment (chavid) [reply CWP/papers/HSF-CWP-2017-02_training/latex/cwp-wg-training-careers.tex]: Well, I was not thinking of the creation of a real journal. As you said, it is a massive amount of work. Perhaps we should investigate where and how the authors may publicize their work. My original think was : a young enough author will not devote too much time to pedagogy and know-how sharing, if he is not rewarded the same as for writing paper. Perhaps, if we have a way to tailor how often a tutorial is used, or a way for students to "likes" the tutorial, an author may feel to cite his "most used and/or liked" tutorials, the same he would cite his papers. Going a little further, investigating how to mimic the publications mechanism, we could imagine that the tutos authors as people taking inspirations in previous existing tutos and rebuild their own one, improved here and there, not forgetting to cite the original tutos... something just like papers. This way, each author stay the chief in HIS tuto, is rewarded accordingly, yet respect and cite his sources of inspiration, and the global quality improve, and the tutos keep updated thanks to new contributors... a model which is not the one of a centralized place with commonly agreed tutorials, but a distributed living model looking like research papers. Not sure I am clear :)

**Comment by chavid** [comment]
I tried a rewrite.

**Comment by graeme-a-stewart** [comment]
Hi @chavid - you said you tried a rewrite but there are no new commits. Did you need to push something to your branch?

**Comment by chavid** [comment]
Bad manoeuver in my branch. Preparing a new puul-request.

**Pull request closed without merging**